visualization_scripts/run.py

```diff
@@ -0,0 +1,175 @@
+""" 
+script loads style file and TPS output file into Cytoscape session
+
+@author: Adam Shedivy
+
+"""
+
+# imports 
+from py2cytoscape.data.cynetwork import CyNetwork
+from py2cytoscape.data.cyrest_client import CyRestClient
+from py2cytoscape.data.style import StyleUtil
+import py2cytoscape.util.cytoscapejs as cyjs
+import py2cytoscape.cytoscapejs as renderer

```

Comment:
I'm getting the following error when running the script in the current conda environment:
```
Traceback (most recent call last):
  File "run.py", line 13, in <module>
    import py2cytoscape.cytoscapejs as renderer
  File "C:\Users\agitter\.conda\envs\py2cyto\lib\site-packages\py2cytoscape\cytoscapejs\__init__.py", line 21, in <module>
    init()
  File "C:\Users\agitter\.conda\envs\py2cyto\lib\site-packages\py2cytoscape\cytoscapejs\__init__.py", line 13, in init
    from IPython.core.display import display, Javascript
ModuleNotFoundError: No module named 'IPython'
```

Can we delete some of these import statements to reduce the dependencies?  Or is IPython a true dependency and we need to expand the environment?  Based on my testing, we should be able to delete this import.

Reply:
I updated my editor settings, and the program does not use networkx, pandas, and json.
I will remove these imports for the first pull request